packages/components/src/GridLayout/Grid.component.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'react-grid-layout/css/styles.css';
+import { Responsive, WidthProvider } from 'react-grid-layout';
+import Tile from './Tile';
+import { SKELETON_TILE_CONF } from './Tile/Skeleton/SkeletonTile.component';
+// eslint-disable-next-line no-unused-vars
+import './Grid.scss';
+
+// eslint-disable-next-line new-cap
+const ResponsiveGridLayout = WidthProvider(Responsive);
+const MARGIN = 20;
+
+const columns = {
+	xs: 2,
+	s: 4,
+	m: 6,
+	l: 12,
+};
+
+const breakpoints = {
+	xs: 479,
```

Comment:
why precisely 479 ?

Reply:
guideline says so

Reply:
In fact the guideline is not really correct, the small breakpoint should be 0 to 768

Reply:
will check with anne